packages/view-types/spatial/src/SpatialOptions.js

```diff
@@ -28,16 +28,16 @@ const ToggleFixedAxisButton = ({
   use3d,
 }) => {
   const classes = useToggleStyles();
-  return (
-    <TableRow>
-      <TableCell className={classes.cameraLabel}>
+  return use3d && (
```

Comment:
Is this change to using rendering vs. the `disabled` prop related to accessibility?

Reply:
It is done for the same reasons as this one: https://github.com/vitessce/vitessce/pull/1612/files#r1266860907 Let me know if I should revert it.

Reply:
I read the comments further and I see that both of you are on the opinion that using `disabled` is better, so I reverted this change in commit e5371cbf